src/common/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { Dispatch } from "react";
+import { updateBTCSpotPriceAction } from "../lib/actions";
+import { PriceActions } from "../lib/types/index";
+import { Option } from "../lib/types/option";
+
+export const updateBTCSpotPrice = async (dispatch: Dispatch<PriceActions>): Promise<void> => {
+    const mockRandom = Math.random() * 100 - 50;
+    const newPrice: number = await new Promise((resolve) => setTimeout(() => resolve(10500 + mockRandom), 500));
```

Comment:
We might need to double this `10500` now :joy:

Reply:
I'm gonna say it helps make it clear that it's a mock, yes, that's definitely why I used that.